toolset/setup/linux/languages/dotnetcore.sh

```diff
@@ -2,11 +2,11 @@
 
 fw_installed dotnetcore && return 0
 
-sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
-sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
+sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 417A0893
 sudo apt-get update
 
-sudo apt-get install dotnet-dev-1.0.0-preview2-003131 -y
+sudo apt-get install dotnet-dev-1.0.0-preview2.1-003177 -y
```

Comment:
This does install the 1.1 runtime.  See https://github.com/dotnet/core/blob/master/release-notes/1.1/1.1.md.  The version number in the package name is the SDK version (1.0.0 Preview 2.1) rather than the runtime version (1.1).